src/overlays/actors/ovl_Bg_Spot18_Obj/z_bg_spot18_obj.c

```diff
@@ -21,47 +41,259 @@ const ActorInit Bg_Spot18_Obj_InitVars = {
     (ActorFunc)BgSpot18Obj_Update,
     (ActorFunc)BgSpot18Obj_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Spot18_Obj/func_808B8910.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Spot18_Obj/func_808B8A5C.s")
+u8 D_808B90F0[] = {
+    0x01,
+    0x01,
+    0x01,
+    0x00,
+};
+
+f32 D_808B90F4[] = {
+    0.1f,
+    0.1f,
+};
+
+UNK_TYPE D_808B90FC[] = {
+    0x06002FE4,
+    0x0600261C,
+};
+
+u32 D_808B9104[] = {
+    0,
+    0,
+};
+
+BgSpot18ObjInitFunc D_808B910C[] = {
+    func_808B8A98,
```

Comment:
I would call this sInitFuncs since we know what it is. other than that, looks good

Reply:
I was going to do that, but there's the other init func array a few symbols down. I could name them 1 and 2, but I wasn't sure..

Reply:
oh true i didnt see that. it will be better to name them when we know what the init funcs are specifically for

Reply:
okay, yeah that's what I thought. sounds good